### ISSUE TITLE
Loggers in RequestsHooks

### ISSUE THREAD

**Issue by visualist** [SoftInstigate/restheart#185]
Hi, this is more of a gotcha for hook-developers, and a question...

tl;dr -- The standard Logger does not work in classes which are not under the org.restheart hierarchy, such as any added singletons (e.g, a RequestHook).

My first steps in exploring how to implement a RequestHook started with the provided example, Snooperhook (org.restheart.hal.metadata.singletons.SnooperHook). I made my own copy of this class, MySnooperHook, and put it in my own namespace, adjusted the package statement, imported org.restheart.hal.metadata.singletons.Hook. It seemed like I was good to go. Everything compiled and nothing complained. But it appeared as if it was never called. Finally, when I set this project up in IntelliJ/CE and debugged it, stepped through the code, I saw it was entering the doesSupportRequests and hook methods, it just wasn't logging anything to the log output. (As a counter example, if I put MySnooperHook back into the org.restheart.hal.metadata.singletons package, it works.)

So my question is this. I'd like to keep my hook class in my own namespace, but I'd also like to have its log output go into the already setup log-output stream. Is there a way to do this?

**Comment by ujibang** [comment]
the logger of restheart is configured to only display messages on org.restheart namespace

you can get your logger as follows:

```
private static final Logger LOGGER = LoggerFactory.getLogger("org.restheart.custom.YourClass");
```

**Comment by visualist** [comment]
Thanks, that worked!
